lua/terrortown/lang/ru.lua

```diff
@@ -1,13 +1,9 @@
 -- Russian language strings
 
--- This localization file is being moderated and constantly updated by Satton(RU).
--- Please, in case you are making some changes ping @Satton2 on GitHub, contact him on Steam (STEAM_0:0:85981967) or Discord (Satton(RU)#5794).
-
 local L = LANG.CreateLanguage("ru")
 
--- Compatibility language name that might be removed soon.
--- the alias name is based on the original TTT language name:
--- https://github.com/Facepunch/garrysmod/blob/master/garrysmod/gamemodes/terrortown/gamemode/lang/russian.lua
```

Comment:
i think a comment like this is actually helpful in the main english translation, sorry to bother you again, but can we add this to every file as a comment?

Reply:
Hmm? Isn't this in every file anyway?

Reply:
No, there is no comment.
I'd just like to keep a small comment explaining that this is the alias to the TTT language name to provide a fallback for when addons register translations for languages referencing those names.

Reply:
ah, I removed it in the russian file but nowhere else, hence my confusion. Dunno why I did that. Probably by mistake